tests/wdio.grid.firefox.conf.ts

```diff
@@ -0,0 +1,18 @@
+// wdio.grid.conf.ts
+// extends the main configuration file to add the selenium grid address
+import { merge } from 'lodash-es';
+import { URL } from 'url';
+
+// @ts-ignore
+import { config as defaultConfig } from './wdio.firefox.conf.ts';
+
+const gridUrl = new URL(process.env.GRID_HOST_URL as string);
+const protocol = gridUrl.protocol.replace(':', '');
+
+export const config = merge(defaultConfig, {
```

Comment:
Do we need a loadash merge? Couldn't we get away with something like:
```
{
... defaultConfig,
protocol,
...
}

```

Reply:
I don't know. The example of inheriting configurations is using a new dependency deepmerge-ts and by Saul's proposal (if I remember correctly), I changed it to use loadash and to avoid introducing a new dependency.  

https://webdriver.io/docs/organizingsuites#inherit-from-main-config-file

Reply:
But do we use the deeply nested override? 

I mean what do we want to happen when we override `protocol` for example? Do we want the results to have the new protocol value or we want a merged object and have the old props of protocol and also the new props?

Reply:
Yeah, you are right. I guess, we can get rid of those merges in the grid file. In the firefox one, we need it as we need it to merge the props and overwrite some of value but not all of them.

Reply:
I would also ve fine to leave it as it is since on the other place is like that. 

Whatever you decide.